setup.py

```diff
@@ -53,5 +53,5 @@
     },
     url='https://github.com/google/xarray-beam',
     packages=setuptools.find_packages(exclude=['examples']),
-    python_requires='>=3',
```

Comment:
Could you please remove the explicit upper bound?

In my experience, pinning an upper bound on dependencies in Python libraries is almost always a mistake: https://iscinumpy.dev/post/bound-version-constraints

Reply:
Thanks for the post. Removed.